.github/workflows/deploy.yaml

```diff
@@ -8,6 +8,7 @@ on:
 permissions:
   # required by aws-actions/configure-aws-credentials
```

Comment:
This is now required by the riff-raff role, instead of `aws-actions/configure-aws-credentials`